src/main/java/org/apache/pirk/query/wideskies/Query.java

```diff
@@ -26,41 +26,51 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.function.Consumer;
 
+import com.google.gson.annotations.Expose;
 import org.apache.pirk.encryption.ModPowAbstraction;
 import org.apache.pirk.serialization.Storable;
 import org.slf4j.Logger;
 import org.slf4j.LoggerFactory;
 
 /**
  * Class to hold the PIR query vectors
- *
  */
+
 public class Query implements Serializable, Storable
```

Comment:
I assume we can also drop ```implements Serializable``` now too, right?
This instance, and friends elsewhere.